tests/test_intake_es.py

```diff
@@ -113,6 +114,21 @@ def test_to_dask(engine):
     assert out[df.columns].equals(df)
 
 
+def test_to_dask_with_partitions(engine):
```

Comment:
Please also add tests for a query which is JSON.

Do you plan for a partitioned sequence, which in dask is called a "bag", and would be constructed with `dask.bag.from_delayed`?

Reply:
By JSON query you mean using the Query DSL right? i will proceed with adding one. 

If I understood correctly the `ElasticSearchSeqSource` ojbect is equivalent to a `JSON` response. Would you like me to add a `to_dask` method here?

Reply:
Correct. The query can either be a string which can be interpreted as JSON (`"{...}"`) or a simple string like `'score:[0 TO 150]'`.

Reply:
Take a look, added `to_dask` in ElasticSearchSeqSource` as well as JSON testcases.